package.json

```diff
@@ -116,6 +117,7 @@
     "@typescript-eslint/parser": "2.3.2",
     "babel-jest": "^25.2.3",
     "babel-loader": "8.0.6",
+    "babel-plugin-macros": "^2.8.0",
     "brotli-webpack-plugin": "^1.1.0",
```

Comment:
wait, double checking wat's this for? i saw `macros` being used as a babel plugin further down, is that how it's being used?

Reply:
ya cos without this babel-plugin-macros, all macros will not work